float32[6] joint_value
